evidence/pool.go

```diff
@@ -520,6 +524,20 @@ func (evpool *Pool) removeEvidenceFromList(
 	}
 }
 
+// flushConsensusBuffer moves the evidence produced from consensus into the evidence pool
+// and list so that it can be broadcasted and proposed
+func (evpool *Pool) flushConsensusBuffer() {
+	for _, ev := range evpool.consensusBuffer {
+		if err := evpool.addPendingEvidence(ev); err != nil {
+			evpool.logger.Error("failed to flush evidence from consensus buffer to pending list: %w", err)
+		}
+
+		evpool.evidenceList.PushBack(ev)
+	}
+	// reset consensus buffer
+	evpool.consensusBuffer = make([]types.Evidence, 0)
```

Comment:
any potential data races?

Reply:
They are at different stages of consensus I believe but it might pay to put a mutex on the slice just in case